library_information_validator.py

```diff
@@ -0,0 +1,318 @@
+# load libraries
```

Comment:
I'm not very good with Python, but I do think it's a good idea to begin the script with:
```
#!/usr/bin/env python
```

Reply:
Yupp, I agree

Reply:
That line tells the computer what program to use when running the file.

Reply:
Just forgot to copy + paste it.